src/iarc7_motion/iarc_tasks/velocity_test_task.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+
+import math
+import rospy
+import tf2_ros
+import tf2_geometry_msgs
+import threading
+
+from geometry_msgs.msg import TwistStamped
+from geometry_msgs.msg import Point
+from geometry_msgs.msg import PointStamped
+from nav_msgs.msg import Odometry
+
+from .abstract_task import AbstractTask
+from iarc_tasks.task_states import (TaskRunning,
+                                    TaskDone,
+                                    TaskCanceled,
+                                    TaskAborted,
+                                    TaskFailed)
+from iarc_tasks.task_commands import (VelocityCommand,
+                                      ArmCommand,
+                                      NopCommand)
+
+from height_holder import HeightHolder
+from height_settings_checker import HeightSettingsChecker
+from acceleration_limiter import AccerlationLimiter
+
+class VelocityTestTaskState:
+    init = 0
+    moving = 1
+    waiting = 2
+
+class VelocityTestTask(AbstractTask):
+
+    def __init__(self, task_request):
+
+        self._drone_odometry = None
+        self._canceled = False
+
+        self._tf_buffer = tf2_ros.Buffer()
+        self._tf_listener = tf2_ros.TransformListener(self._tf_buffer)  
+
+        self._lock = threading.RLock()
+
+        self._current_velocity_sub = rospy.Subscriber(
+            '/odometry/filtered', Odometry,
+            self._current_velocity_callback)
+
+        try:
+            self._TRANSFORM_TIMEOUT = rospy.get_param('~transform_timeout')
+            self._MAX_TRANSLATION_SPEED = rospy.get_param('~max_translation_speed')
+            self._MAX_Z_VELOCITY = rospy.get_param('~max_z_velocity')
+            self._TRANSLATION_SPEED_X = rospy.get_param('~test_velocity_x')
```

Comment:
I don't think these should be parameters. These should be passed in through the action request.